Change in IMU message type

Running under Kinetic I see the following message from `roslaunch jackal_base base.launch `:

```
[ERROR] [1536651743.105581726]: Client [/imu_filter] wants topic /imu/mag to have datatype/md5sum [sensor_msgs/MagneticField/2f3b0b43eed0c9501de0fa3ff89a45aa], but our version has [geometry_msgs/Vector3Stamped/7b324c7325e683bf02a9b14b01090ec7]. Dropping connection.
```

This is due to a change in [imu_filter_madgwick](http://wiki.ros.org/imu_filter_madgwick#Supported_devices):

```
~use_magnetic_field_msg (bool, default: false for Hydro and Indigo, true for Jade+)

    If set to true, subscribe to the /imu/mag topic as a sensor_msgs/MagneticField; if set to false (deprecated), use geometry_msgs/Vector3Stamped. 
```

Therefore for ROS distros below Jade `base.launch` requires the following added to the `imu_filter` node configuration:
```
          <param name="use_magnetic_field_msg" value="false" />
```

--- a/jackal_base/launch/base.launch
+++ b/jackal_base/launch/base.launch
@@ -34,6 +34,7 @@
       zeta: 0.001
       publish_tf: false
     </rosparam>
+    <param name="use_magnetic_field_msg" value="false" />
   </node>
 
   <!-- Differential controller and basic localization -->

